crates/node/builder/src/rpc.rs

```diff
@@ -697,30 +700,18 @@ where
 }
 
 /// Builder for engine API RPC module.
-///
-/// This builder type is responsible for providing an instance of [`IntoEngineApiRpcModule`], which
-/// is effectively a helper trait that provides the type erased [`jsonrpsee::RpcModule`] instance
-/// that contains the method handlers for the engine API. See [`EngineApi`] for an implementation of
-/// [`IntoEngineApiRpcModule`].
```

Comment:
why remove the docs

Reply:
sure, ofc i'll add this back and look at review comment